about.html

```diff
@@ -0,0 +1,142 @@
+          AUGMENTED REALITY ART SUMMIT 2022
+        </h1>
+      </header>
+      <div class="intro-msg">
+        <p>As a year of technological shifts, 2022 changed the art and metaverse industry with new trends, synergies, and key players. Creatives increasingly understood the importance of collaboration, going beyond the obvious and using new mediums in their work, such as AR, VR, NFTs, etc. The AR Art Summit was born as a reflection of the current shifts happening in the art world.
+        <p id="second-para">Creatives increasingly understood the importance of collaboration, going beyond the obvious and using new mediums in their work, such as AR, VR, NFTs, etc. The AR Art Summit was born as a reflection of the current shifts happening in the art world.
+        </p>
+        </p>
+      </div>
+      <div class="contact-info">
+        <div class="info-msg">
+          <p>Please contact us per Email for any further questions about AR Art Summit 2022!</p>
+        </div>
+        <div class="email-info">
+          <p>arartsummit2022@arkorea.org</p>
+        </div>
+      </div>
+    </section>
+    <section id="logo-sctn">
+      <header class="logo-headr">
+        <h2>AR Art Summit 2022 Logo</h2>
+        <div class="orange-bar"></div>
+      </header>
+      <div class="logo-info">
+        <p>The logo of AR Art Summit 2022 was decided through the logo competition from 8 June to 7 July.</p>
+      </div>
+      <div id="logo-image">
+        <img src="imgs/ar-art-summit-2022-600x315.jpeg" alt="summit logo">
+      </div>
+      <br>
+    </section>
+    <section class="past-summits">
+      <header class="past-header">
+        <h2>See the Past AR Art Summits</h2>
+        <div class="orange-bar"></div>
+      </header>
+      <div class="past-sub">
+        <p>Take a look at the last two AR Art Summits which took place in Beunos Aires and in Warsaw.</p>
+      </div>
+      <div class="about-years">
+        <div class="about-2013">
+          <img src="imgs/about_2013_hover.png" alt="2 men in 2013 summit">
+        </div>
+        <div class="about-2011">
+          <img src="imgs/about_2011_hover.png" alt="groupie in 2011 summit">
+        </div>
+      </div>
+    </section>
+    <section class="partner">
+      <header class="partner-header">
+        <h2>Partners</h2>
+        <div class="orange-bar"></div>
+      </header>
+      <div class="sponsor-group">
+        <div class="sponsors">
+          <img src="imgs/mozilla.png" alt="">
+        </div>
+        <div class="sponsors">
+          <img src="imgs/googlekorea-removebg-preview.png" alt="">
+        </div>
+        <div class="sponsors">
+          <img src="imgs/naver-removebg-preview.png" alt="">
+        </div>
+        <div class="sponsors">
+          <img src="imgs/kakao-removebg-preview.png" alt="">
+        </div>
+        <div class="sponsors">
+          <img src="imgs/airbnb-removebg-preview.png" alt="">
+        </div>
```

Comment:
- [ ]  all of your pages should be accessible, currently, you have some empty alt attributes kindly add meaningful alts for each img tag you have